src/winetricks

```diff
@@ -9439,6 +9439,34 @@ verify_dotnet48()
 
 #----------------------------------------------------------------
 
+w_metadata dotnetcore2 dlls \
+    title="MS .NET Core Runtime 2.1 LTS" \
+    publisher="Microsoft" \
+    year="2020" \
+    media="download" \
+    conflicts="" \
```

Comment:
Nitpick, but you don't need to define this if there's not a conflict (but good on you for paying attention ;)).